src/planner/FirstSupportedMetaPlanner.cpp

```diff
@@ -0,0 +1,22 @@
+#include "aikido/planner/FirstSupportedMetaPlanner.hpp"
+
+namespace aikido {
+namespace planner {
+
+//==============================================================================
+trajectory::TrajectoryPtr FirstSupportedMetaPlanner::plan(
+    const Problem& problem, Result* result)
+{
+  for (const auto& planner : mPlanners)
+  {
+    if (planner->canSolve(problem))
+      return planner->plan(problem, result);
+  }
+
+  // TODO: Set result "no supported planner".
```

Comment:
(bringing to your addition)

Reply:
I believe we will go over all the planner implementation including meta planners after this PR is merged.